docker/Dockerfile.local

```diff
@@ -1,4 +1,5 @@
-FROM purduearc/rocket-league
+ARG TAG=local
+FROM purduearc/rocket-league:${TAG}
```

Comment:
Is this meant to be here, or just for local testing?

Reply:
Typo, line 1 is supposed to be latest. So you can run ./docker-run --build-arg TAG=69 and the docker image for this PR